src/main.js

```diff
@@ -113,4 +118,36 @@ form.addEventListener('submit', (e) => {
     email.style.border = '3px solid red';
     error.textContent = 'X   Email should be in lowerCase';
   }
-});
\ No newline at end of file
+});
+if (storageAvailable('localStorage')) {
+  const setFormValues = () => {
+    const formData = {
+      name: form.contact_name.value,
+      email: form.contact_email.value,
+      message: form.contact_message.value,
+    };
+    localStorage.setItem('formData', JSON.stringify(formData));
+  };
+
+  form.contact_name.addEventListener('change', setFormValues);
+  form.contact_email.addEventListener('change', setFormValues);
+  form.contact_message.addEventListener('change', setFormValues);
+}
+const checlocal = () => {
+  let name = '';
+  let email = '';
+  let message = '';
+  if (JSON.parse(localStorage.getItem('formData')) === null) {
+    name = null;
+    email = null;
+    message = null;
+  } else {
+    name = JSON.parse(localSotrage.getItem('formData')).name;
+    email = JSON.parse(localSotrage.getItem('formData')).email;
+    message = JSON.parse(localSotrage.getItem('formData')).message;
+  }
+  CONTACT_NAME.value = name;
+  CONTACT_EMAIL.value = email;
+  CONTACT_MESSAGE.value = message;
+};
```

Comment:
- [ ] When the user loads the page if there is any data in the local storage the input fields are pre-filled with this data. Please Implement it.
`

Reply:
thank you I will fix it